packages/expo-font/src/__tests__/Font-test.ts

```diff
@@ -17,286 +27,330 @@ afterEach(() => {
+      expect(Font.isLoading('test-font')).toBe(false);
+
+      const mockAsset2 = _createMockAsset();
+      await Font.loadAsync('test-font', mockAsset2);
+      expect(NativeFontLoader.loadAsync).toHaveBeenCalledTimes(1);
+      expect(Font.isLoaded('test-font')).toBe(true);
+      expect(Font.isLoading('test-font')).toBe(false);
+    });
 
-    const mockAsset1 = _createMockAsset();
-    const loadPromise1 = Font.loadAsync('test-font', mockAsset1);
-    expect(Font.isLoaded('test-font')).toBe(false);
-    expect(Font.isLoading('test-font')).toBe(true);
+    it(`coalesces concurrent loads`, async () => {
+      const NativeFontLoader = NativeModulesProxy.ExpoFontLoader;
 
-    const mockAsset2 = _createMockAsset();
-    const loadPromise2 = Font.loadAsync('test-font', mockAsset2);
-    expect(Font.isLoaded('test-font')).toBe(false);
-    expect(Font.isLoading('test-font')).toBe(true);
+      const mockAsset1 = _createMockAsset();
+      const loadPromise1 = Font.loadAsync('test-font', mockAsset1);
+      expect(Font.isLoaded('test-font')).toBe(false);
+      expect(Font.isLoading('test-font')).toBe(true);
 
-    await Promise.all([loadPromise1, loadPromise2]);
-    expect(NativeFontLoader.loadAsync).toHaveBeenCalledTimes(1);
-    expect(Font.isLoaded('test-font')).toBe(true);
-    expect(Font.isLoading('test-font')).toBe(false);
-  });
+      const mockAsset2 = _createMockAsset();
+      const loadPromise2 = Font.loadAsync('test-font', mockAsset2);
+      expect(Font.isLoaded('test-font')).toBe(false);
+      expect(Font.isLoading('test-font')).toBe(true);
 
-  it(`rejects all coalesced loads`, async () => {
-    const mockAsset1 = {
-      downloaded: false,
-      downloadAsync: jest.fn(async () => {}),
-    };
-    const loadPromise1 = Font.loadAsync('test-font', mockAsset1);
-    expect(Font.isLoaded('test-font')).toBe(false);
-    expect(Font.isLoading('test-font')).toBe(true);
-
-    const mockAsset2 = _createMockAsset();
-    const loadPromise2 = Font.loadAsync('test-font', mockAsset2);
-    expect(Font.isLoaded('test-font')).toBe(false);
-    expect(Font.isLoading('test-font')).toBe(true);
-
-    await expect(loadPromise1).rejects.toBeDefined();
-    await expect(loadPromise2).rejects.toBeDefined();
-    expect(Font.isLoaded('test-font')).toBe(false);
-    expect(Font.isLoading('test-font')).toBe(false);
-  });
+      await Promise.all([loadPromise1, loadPromise2]);
+      expect(NativeFontLoader.loadAsync).toHaveBeenCalledTimes(1);
+      expect(Font.isLoaded('test-font')).toBe(true);
+      expect(Font.isLoading('test-font')).toBe(false);
+    });
 
-  it(`accepts a map of fonts to multi-load`, async () => {
-    await Font.loadAsync({
-      'test-font-1': _createMockAsset({
-        localUri: 'file:/test/test-font-1.ttf',
-      }),
-      'test-font-2': _createMockAsset({
-        localUri: 'file:/test/test-font-2.ttf',
-      }),
+    it(`rejects all coalesced loads`, async () => {
+      const mockAsset1 = {
```

Comment:
Prefer calling `_createMockAsset` when possible so that we can consolidate asset creation code there